pycheribuild/projects/cross/cheribsd.py

```diff
@@ -1353,8 +1353,8 @@ def setup_config_options(cls, **kwargs):
 # FIXME: this should inherit from BuildCheriBSD to avoid subtle problems
 class BuildCheriBsdMfsKernel(SimpleProject):
     project_name = "cheribsd-mfs-root-kernel"
-    dependencies = ["disk-image-minimal"]
-    # TODO: also support building a non-CHERI kernel... But that needs a plain MIPS disk-image-minimal first...
+    dependencies = ["disk-image-mfs-root"]
+    # TODO: also support building a non-CHERI kernel... But that needs a plain MIPS disk-image-mfs-root first...
```

Comment:
I think this has long been resolved? Since you're changing this line might as well remove it?

Reply:
Oh, hm, I had interpreted that as meaning FreeBSD not CheriBSD-without-CHERI, but yes, this can go